src/main/java/com/autotune/utils/AnalyzerConstants.java

```diff
@@ -420,4 +420,31 @@ public static final class PerformanceProfileConstants {
                 "advance-cluster-management", "AdvancedClusterMgmtImpl"
         );
     }
+
+    public enum K8S_OBJECT_TYPES {
+        deployment,
+        deploymentConfig,
+        statefulset,
+        replicaset,
+        replicationController,
+        daemonset,
+    }
+
+    public static final class K8sObjectConstants {
+        private K8sObjectConstants() {
+
+        }
+        public static final class Types {
+            private Types() {
+
+            }
+
+            public static final String DEPLOYMENT = "deployment";
```

Comment:
@dinogun Can I please know if these are the right formats to compare the k8s object types which we receive in JSON? Please lemme know if I need to make any casing changes.

Reply:
@kusumachalasani can comment what is returned by the queries and we can keep it to be the same as those returned by the queries to avoid having to manipulate it later

Reply:
@bharathappali Below are the workloads and their format(based on the output from queries) that are supported for MVP.
```
deployment
deploymentconfig
daemonset
statefulset
replicaset
replicationcontroller
```

Reply:
Thanks @kusumachalasani 

@dinogun I have implemented `equals ignore case` to confirm the match. Should we implement a strict match?
I'm more inclined towards `equals ignore case` coz things may change in future and the code still works unaffected.